File: 1833-find-the-highest-altitude/find-the-highest-altitude.c
int largestAltitude(int* gain, int gainSize) {
  int cur=0;
  int max=0;
  for(int i=0;i<gainSize;i++){
    cur = cur+gain[i];
    if(cur>max){
        max=cur;
    }
  } 
  return max; 
}

File: 1833-find-the-highest-altitude/find-the-highest-altitude.py
class Solution(object):
    def largestAltitude(self, gain):
        """
        :type gain: List[int]
        :rtype: int
        """
        i=0
        j=0
        for g in gain:
            i += g
            j = max(j,i)
        return j
        